From 56c865e2a7269d63f0780b4cd1ed6fdb18e3ff1b Mon Sep 17 00:00:00 2001
From: Changqing Li <changqing.li@windriver.com>
Date: Wed, 26 Sep 2018 17:41:59 +0800
Subject: [PATCH] fix do_package_qa error

fix below error:
do_package_qa: QA Issue: ELF binary 'xxx/usr/bin/docker' 
has relocations in .text [textrel]

Upstream-Status: Inappropriate [oe-specific]

Signed-off-by: Changqing Li <changqing.li@windriver.com>
---
 cli/scripts/build/dynbinary | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cli/scripts/build/dynbinary b/cli/scripts/build/dynbinary
index 3c32ed3..938e086 100755
--- a/cli/scripts/build/dynbinary
+++ b/cli/scripts/build/dynbinary
@@ -9,6 +9,6 @@ source ./scripts/build/.variables
 
 echo "Building dynamically linked $TARGET"
 export CGO_ENABLED=1
-go build -o "${TARGET}" -tags pkcs11 --ldflags "${LDFLAGS}" "${SOURCE}"
+go build -o "${TARGET}" -tags pkcs11 --ldflags "${LDFLAGS}" -buildmode=pie "${SOURCE}" 
 
 ln -sf "$(basename "${TARGET}")" build/docker
-- 
2.7.4

